src/gateway/kong-production/monitoring/statsd.md

```diff
@@ -1,6 +1,95 @@
 ---
-title: Monitoring with Statsd
-
+title: Monitoring with StatsD
+content_type: how-to
 ---
 
-## PLACEHOLDER How to Monitor with Statsd
\ No newline at end of file
+[StatsD](https://github.com/statsd/statsd) is a network daemon that collects
+and aggreates performance metrics by listening on the network for simple 
+text based statistics data published by applications. 
+
+This guide will help you setup a test {{site.base_gateway}} and
+StatsD service. Then you will generate sample requests to {{site.base_gateway}} and
+observe the collected monitoring data. 
+
+### Prerequisites
+* A test {{site.base_gateway}} instance is installed using the following script:
+   ```sh
+   curl -Ls get.konghq.com/quickstart | sh -s -- -m
+   ```
+   The `-m` flag instructs the script to install a mock service that is used in this guide to generate sample metrics.
+   {:.note}
+   > **Note:** This guide's instructions use a test {{site.base_gateway}} instance to demonstrate how the StatsD plugin can be used to collect metrics. If you want to use an existing {{site.base_gateway}} instance for this guide, you must modify the connection information in the commands.
```

Comment:
@rspurgeon @lena-larionova I'm not sure why GitHub removed our conversation, but I added this and was wondering what you think. Should we provide additional information in the note about what services/routes/etc. they should add to an existing instance, or would users know enough to know what to modify in the commands?

Reply:
I'm in favor of this.  I do think if someone is going to be connecting StatsD, from inside Docker, to an existing Gateway, they likely need to have enough knowledge to do that without a lot of very specific guidance from them.  Their gateway could be anywhere, so we can't give them exact guidance.

The only other change I might make is, instead of it being a note after the user runs the script, but this guidance before the script and tell them it's optional and why.  So i'd move this up to the instructions for Step 1

Reply:
I didn't notice that this step got moved to Prerequisites.  IMO this isn't a prereq, it's a step in the guide.  We may choose to make that step optional, but I think prereqs are tools that must be installed or other system setups that are required to proceed.    If this were a pre-req, it would need to come after Docker or curl becuase it uses both of those tools.  I still very much prefer that this step be part of the guide with text to make it optional if desired.